CMakeLists.txt

```diff
@@ -1129,7 +1129,6 @@ ENDIF()
 # version has the necessary patches to make it work.
 SET(SYSTEM_LIBRARIES
   CURL
-  EDITLINE
```

Comment:
This means `-DEDITLINE=system` won't be used when '-DWITH_SYSTEM_LIBRARIES=ON' is specified. Why so?